tensorboard/backend/application.py

```diff
@@ -177,8 +179,9 @@ def standard_tensorboard_wsgi(flags, plugin_loaders, assets_zip_provider):
         start_reloading_multiplexer(
             multiplexer, path_to_run, reload_interval, flags.reload_task
         )
+
     return TensorBoardWSGIApp(
-        flags, plugin_loaders, data_provider, assets_zip_provider, multiplexer
+        flags, plugin_loaders, data_provider, assets_zip_provider, multiplexer,
```

Comment:
sorry, forgot to mention earlier: can we revert the churn to these two
lines? (Black should be fine with it either way, so there’s no need to
pollute the diff and blame.)